cdap-e2e-tests/src/e2e-test/features/systemadmin/SysAdminDesignTime.feature

```diff
@@ -0,0 +1,62 @@
+#
+# Copyright © 2023 Cask Data, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+
+@Sysadmin
+Feature: Sysadmin - Validate system admin page design time scenarios
+
+  @Sysadmin
+  Scenario:Validate user is able to create new system preferences and able to delete the added system preferences successfully
+    Given Open Datafusion Project to configure pipeline
+    Then Click on the "System Admin" link from menu
+    Then Click on the Configuration link on the System admin page
+    Then Select "systemPreferences" option from Configuration page
+    Then Click on edit system preferences to set system preferences
+    Then Enter key value pairs to set preferences: "keyValue" with values from json: "systemPreferences1"
+    Then Click on Save and Close button to save the preferences
```

Comment:
``` Click on the Save preferences button ``` or ``` Save the preferences```

Reply:
The button name is 'Save & Close' shall I add the suggested step?

Reply:
Changed the step definition